app/views/topup/af-in-consultation.html

```diff
@@ -0,0 +1,157 @@
+{% extends "layout.html" %}
+
+{% block head %}
+  <link href="/public/stylesheets/unbranded.css" media="screen" rel="stylesheet" type="text/css">
+  <link href="/public/stylesheets/phe-nsc-alpha.css" media="screen" rel="stylesheet" type="text/css">
+{% endblock %}
+
+{% block content %}
+
+<div class="govuk-width-container">
+  <!-- <img src="{{ asset_path }}images/phe_nsc_logo.jpg" alt="UK NSC logo" style="height: 42px; border-left: 3px solid #00B092; padding-left: 7px;"> -->
+  <div class="govuk-breadcrumbs">
+    <ol class="govuk-breadcrumbs__list">
+      <li class="govuk-breadcrumbs__list-item">
+        <img src="{{ asset_path }}images/nsc-logo.png" alt="UK NSC logo" style="height: 42px; position: relative; bottom:23px">
+        <a class="govuk-breadcrumbs__link" href="#" style="position: relative; bottom: 26px">
+          UK National Screening Committee
+        </a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="#">Recommendations</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item" aria-current="page">Atrial Fibrillation</li>
+    </ol>
+  </div>
+
+
+  <main class="govuk-main-wrapper">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-xl">
+            <span class="govuk-caption-xl">Adult screening programme</span>
+              Atrial Fibrillation
+          </h1>
+      </div>
+
+      <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-heading-l">Recommendation</h2>
+        <p class="govuk-body govuk-!-font-weight-bold recommendation-badge">
+          <span id="not-recommended">Screening is not currently recommended for this condition</span>
+        </p>
+
+        <div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
```

Comment:
This `!` element turns out looking a bit off-centre to me on Chrome: 

<img width="677" alt="image" src="https://user-images.githubusercontent.com/1370570/57860940-a7f02b00-77ed-11e9-839a-deb22ce8cc0a.png">

Is that unexpected?